closed/autoconf/custom-hook.m4

```diff
@@ -574,7 +574,7 @@ AC_DEFUN([OPENJ9_PLATFORM_SETUP],
 
   AC_SUBST(OPENJ9_BUILDSPEC)
   AC_SUBST(OPENJ9_PLATFORM_CODE)
-  AC_SUBST(COMPILER_VERSION_STRING)
+  AC_SUBST(CXX_VERSION_STRING)
   AC_SUBST(OPENJ9_LIBS_SUBDIR)
   AC_SUBST(OMR_MIXED_REFERENCES_MODE)
```

Comment:
While you're here, could you put these in alphabetical order?

Reply:
Done.